src/analytics/PosthogAnalytics.ts

```diff
@@ -297,7 +296,7 @@ export class PosthogAnalytics {
     const posthogIdMaterial = "ec" + accountAnalyticsId + client.getUserId();
     const bufferForPosthogId = await crypto.subtle.digest(
       "sha-256",
-      Buffer.from(posthogIdMaterial, "utf-8"),
+      new TextEncoder().encode(posthogIdMaterial),
```

Comment:
is this needed for the yarn berry switch or just randomly here?

Reply:
I was trying out Yarn's plug-n-play mode, and it needed this change. I reverted to the normal node_modules linker so I don't think it's strictly necessary. Would you prefer I exclude it?

Reply:
No I am fine keeping it like this. Just wanted to know its not accidental.

Reply:
Can we (did you already?) make sure we use `TextEncoder` everywhere?